tests/pyfunc/test_model_export_with_class_and_artifacts.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, predict_fn):
 
         def load_context(self, context):
             super().load_context(context)
-            # pylint: disable=attribute-defined-outside-init
```

Comment:
`attribute-defined-outside-init` is a useful rule to prevent undefined attributes.